src/components/Captcha.vue

```diff
@@ -12,6 +12,7 @@ export default {
     props: ['network'],
     data() {
         return {
+            recaptchaURL: '',
```

Comment:
`recaptchaURL` should be `recaptchaUrl`. Acronyms should always be in camel case if over 2 chars long